MetadataExtractor/Formats/Wav/WavRiffHandler.cs

```diff
@@ -0,0 +1,59 @@
+#region License
+//
+// Copyright 2002-2019 Drew Noakes
+// Ported from Java to C# by Yakov Danilov for Imazen LLC in 2014
```

Comment:
Are these comments accurate?

Reply:
@kwhopper thanks for pointing this out. I know I commented on these on your PRs. Rather than point it out on all PRs , I'm going to simplify the license banner at the top so it's just a single line that points to more information in the LICENSE file.

Reply:
@kwhopper see #216